File: java-web-spring-boot-3/src/main/java/com/vndevteam/javawebspringboot3/infrastructure/exception/error/ErrorCode.java
package com.vndevteam.javawebspringboot3.infrastructure.exception.error;

import lombok.Getter;

@Getter
public class ErrorCode {
    public static final String SYSTEM_ERROR = "100";
    public static final String REQUEST_METHOD_INVALID = "101";
    public static final String MEDIA_TYPE_INVALID = "102";
    public static final String VALIDATION_INVALID = "103";
    public static final String REQUEST_HEADER_INVALID = "104";
}

File: kotlin-web-spring-boot-3/src/main/kotlin/com/vndevteam/kotlinwebspringboot3/infrastructure/exception/error/ErrorCode.kt
package com.vndevteam.kotlinwebspringboot3.infrastructure.exception.error

class ErrorCode {
    companion object {
        const val SYSTEM_ERROR = "100"
        const val REQUEST_METHOD_INVALID = "101"
        const val MEDIA_TYPE_INVALID = "102"
        const val VALIDATION_INVALID = "103"
        const val REQUEST_HEADER_INVALID = "104"
    }
}
